metadata-ingestion/src/datahub/ingestion/source/common/subtypes.py

```diff
@@ -15,6 +15,7 @@ class DatasetSubTypes(str, Enum):
     SALESFORCE_CUSTOM_OBJECT = "Custom Object"
     SALESFORCE_STANDARD_OBJECT = "Object"
     POWERBI_DATASET_TABLE = "PowerBI Dataset Table"
+    BIGQUERY_TABLE_SNAPSHOT = "Bigquery Table Snapshot"
```

Comment:
A snapshot table is just a table. Don't think we need another syubtype here BIGQUERY_TABLE_SNAPSHOT

Reply:
I actually disagree here - I think "Table Snapshot" would be more clear and it's sufficiently different to merit a separate type